docs/app/Examples/collections/Table/Variations/TableFixedLine.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { Table } from 'stardust'
+
+const TableFixedLine = () => {
+  return (
+    <Table celled fixed singleLine>
+      <Table.Header>
+        <Table.Row>
+          <Table.HeaderCell>Name</Table.HeaderCell>
+          <Table.HeaderCell>Status</Table.HeaderCell>
+          <Table.HeaderCell>Description</Table.HeaderCell>
+        </Table.Row>
+      </Table.Header>
+
+      <Table.Body>
+        <Table.Row>
+          <Table.Cell>John</Table.Cell>
+          <Table.Cell>Approved</Table.Cell>
+          <Table.Cell
+            title="This is much too long to fit I'm sorry about that"
+          >
+            This is much too long to fit I'm sorry about that
```

Comment:
Let's make this content a little longer so it wraps.  On a 15" MacBook it is not quite long enough for `...` to show up:

![image](https://cloud.githubusercontent.com/assets/5067638/18820924/81b7ae10-8356-11e6-88cc-f27c3228d50c.png)